Error after cloning preventing to run the expo app : Can't resolve 'react-native-screens'

I did the following : 

```
git clone git@github.com:callstack/react-native-paper-login-template.git
npm i 
expo start --web
```

## Expected

I expect the app to run

## Result

```bash
 web  Failed to compile.
/home/dka/workspace/github.com/callstack/react-native-paper-login-template/node_modules/react-navigation-stack/lib/module/views/StackView/StackViewCard.js
Module not found: Can't resolve 'react-native-screens' in '/home/dka/workspace/github.com/callstack/react-native-paper-login-template/node_modules/react-navigation-stack/lib/module/views/StackView'
```

--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
     "react-native-gesture-handler": "1.3.0",
     "react-native-paper": "^3.0.0",
     "react-native-status-bar-height": "^2.4.0",
+    "react-native-screens": "^2.2.0",
     "react-native-web": "^0.11.7",
     "react-navigation": "^4.0.10",
     "react-navigation-stack": "^1.10.2"

